policyengine_us/variables/gov/states/az/tax/income/deductions/net_capital_gains/az_net_capital_gains_subtraction.py

```diff
@@ -0,0 +1,22 @@
+from policyengine_us.model_api import *
+
+
+class az_net_capital_gains_subtraction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona net capital gains subtraction"
+    unit = USD
+    documentation = "https://azdor.gov/sites/default/files/2023-03/FORMS_INDIVIDUAL_2022_140i.pdf#page=31"
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.az.tax.income.deductions.net_capital_gains
+
+        long_term_capital_gains = add(
+            tax_unit, period, ["long_term_capital_gains"]
```

Comment:
long-term capital gains are multiplied by the rate but are capital gains in general included in the subtraction? Please double check form and law and add a comment in the .py code